module/config.mjs

```diff
@@ -347,24 +347,78 @@ preLocalize( "lpSpendingTypes" );
  */
 ED4E.testTypes = {
 	arbitrary: {
-		label: "X.arbitraryStepRoll",
-		flavorTemplate: "systems/ed4e/templates/dice/chat-flavor/arbitrary-roll-flavor.hbs",
+		label: "ED.Rolls.Labels.arbitraryTestRoll",
+		flavorTemplate: "systems/ed4e/templates/chat/chat-flavor/arbitrary-roll-flavor.hbs",
 	},
 	action: {
-		label: "X.actionTestRoll",
-		flavorTemplate: "systems/ed4e/templates/dice/chat-flavor/action-roll-flavor.hbs",
+		label: "ED.Rolls.Labels.actionTestRoll",
+		flavorTemplate: "systems/ed4e/templates/chat/chat-flavor/ability-roll-flavor.hbs",
 	},
 	effect: {
-		label: "X.effectTestRoll",
-		flavorTemplate: "systems/ed4e/templates/dice/chat-flavor/effect-roll-flavor.hbs",
+		label: "ED.Rolls.Labels.effectTestRoll",
+		flavorTemplate: "systems/ed4e/templates/chat/chat-flavor/effect-roll-flavor.hbs",
 	},
 	damage: {
-		label: "X.damageTestRoll",
-		flavorTemplate: "systems/ed4e/templates/dice/chat-flavor/damage-roll-flavor.hbs",
+		label: "ED.Rolls.Labels.damageRoll",
+		flavorTemplate: "systems/ed4e/templates/chat/chat-flavor/damage-roll-flavor.hbs",
 	},
 }
 preLocalize( "testTypes", { key: "label" } );
 
+
+/**
+ * The available sub-types of (roll) tests for {@link EdRollOptions}.
+ * @enum {string}
+ */
+ED4E.rollTypes = {
+	ability: {
+		label: "ED.Rolls.Labels.abilityRoll",
+		flavorTemplate: "systems/ed4e/templates/chat/chat-flavor/ability-roll-flavor.hbs",
+	},
+	arbitrary: {
```

Comment:
why do we need the test types as roll types as well? That's not really consistent then. RollTypes should always be a subset of Test Type

Reply:
I agree for arbitrary and damage, we should have 
test types =
action
effect
arbitrary

and everything else should be rollType

Reply:
may be we should find another name for the rolltype effect, because:

testType Effect covers damage rolls, initiative, and effect rolls

and the rolltype effect covers every effect of abilities, like damage rolls but e.g. not inititative